backends/tfhe-cuda-backend/src/cuda_bind.rs

```diff
@@ -311,6 +311,40 @@ extern "C" {
         num_samples: u32,
     );
 
+    /// This scratch function allocates the necessary amount of data on the GPU for
+    /// the public function packing keyswitch implementation on 64-bit
+    pub fn scratch_fp_keyswitch_lwe_list_to_glwe_64(
+        stream: *mut c_void,
+        gpu_index: u32,
+        fp_ks_buffer: *mut *mut i8,
+        glwe_dimension: u32,
+        polynomial_size: u32,
+        input_lwe_ciphertext_count: u32,
+        allocate_gpu_memory: bool,
+    );
+
+    /// Perform public functional packing keyswitch on a vector of 64-bit LWE ciphertexts
+    pub fn cuda_fp_keyswitch_lwe_list_to_glwe_64(
+        stream: *mut c_void,
+        gpu_index: u32,
+        glwe_array_out: *mut c_void,
+        lwe_array_in: *const c_void,
+        fp_ksk_array: *const c_void,
+        fp_ks_buffer: *mut i8,
```

Comment:
is i8 the type chosen for a byte buffer ? I would have thought u8 is more classic

Reply:
Yep, we've been using i8. Do you think we should change to u8 or it doesn't matter?

Reply:
I believe it should not matter but it’s not very standard from my own experience, serialization returns u8 arrays in rust e.g.

Reply:
I will start a conversation within the GPU team about that! Thx.